### PR TITLE
Remove slowlog level

### PR DESCRIPTION
**Pull request by pgomulka**
Setting a slow log level requires an unnecessary conditional logic in `SearchSlowLog` and `IndexingSlowLog`
The behaviour of setting a level on a slow logger can be achieved with correct slow log threshold settings. 
This PR is removing slow log and modifies tests to achieve the same behaviour with changing threshold.
relates https://github.com/elastic/elasticsearch/issues/56171

### CODE REVIEW

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 1]
```diff
@@ -18,4 +18,11 @@ Mapping::
 Deprecations::
 * Remove undocumented endpoints of hot threads API {pull}55109[#55109]
 
+Slow loggers::
+* `index.indexing.slowlog.level` and `index.indexing.slowlog.level` are removed. These settings can be worked around
```
Comment (rjernst) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:22]: It looks like the settings listed here are the same? One should be the search slowlog level?

**Review by rjernst — APPROVED** [review]
LGTM.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 1 above)
Comment (pgomulka) [reply docs/reference/release-notes/8.0.0-alpha1.asciidoc:22]: yes, good find.

**Comment by pgomulka** [comment]
@elasticmachine upgrade branch

**Comment by pgomulka** [comment]
@elasticmachine update branch

**Pull request merged**